daemon/mgr/container.go

```diff
@@ -314,11 +314,7 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		Config:     &config.ContainerConfig,
 		Created:    time.Now().UTC().Format(utils.TimeLayout),
 		HostConfig: config.HostConfig,
-	}
-
-	// validate log config
-	if err := mgr.validateLogConfig(container); err != nil {
-		return nil, err
+		Root:       mgr.ContainerRoot(id),
```

Comment:
Please consider the migration. If the container is existing, the `Root` must be empty. I think we can use the util function to return the path. make senses?